jwst/datamodels/schemas/multispec.schema.yaml

```diff
@@ -118,24 +118,24 @@ allOf:
             type: integer
             fits_keyword: INT_NUM
             fits_hdu: EXTRACT1D
-          time_scale:
+          time_sys:
             title: "principal time system for time-related keywords"
             type: string
             default: "UTC"
             fits_keyword: TIMESYS
             fits_hdu: EXTRACT1D
-          start_utc:
-            title: "UTC at start of integration [MJD]"
+          start_time_mjd:
```

Comment:
`start_time_mjd` and the others are also in the core schema under the same name and same FITS keyword name.  Here they are just mapped to the EXTRACT1D HDU.  I think this is fine, but you had mentioned in another context that this was undesirable (RA_TARG, DEC_TARG).  Thoughts?

Reply:
I didn't think these keywords (MJD-BEG, etc.) were used elsewhere, but maybe I just didn't see them.  These values are copied from the INT_TIMES table data to keywords in the extract_1d extension headers.

Reply:
You're right that they weren't used previously, but there was a recent (external) request to add them to the SCI extension headers of all products, because they conform to the FITS time standard. They have the same values as our EXPSTART, EXPMID, EXPEND keywords, but those keywords don't conform to the standard. So now that the MJD-XXX keywords have been added to the core schema for SCI extensions, I wanted to make the definitions consistent in all places.

Reply:
The entry in core.schema places them in SCI extension headers, while this one puts them in EXTRACT1D extension headers. And since you never have both kinds of extensions in the same file, the keywords will only appear once.